apps/microtvm/zephyr/template_project/microtvm_api_server.py

```diff
@@ -680,12 +695,7 @@ def build(self, options):
                 st.st_mode | stat.S_IEXEC,
             )
 
-        check_call(["cmake", "-GNinja", ".."], cwd=BUILD_DIR, env=env)
-
-        args = ["ninja"]
-        if verbose:
-            args.append("-v")
-        check_call(args, cwd=BUILD_DIR, env=env)
+        check_call(options["west_cmd"].split(" ") + ["build"], cwd=API_SERVER_DIR, env=env)
```

Comment:
Are you using `split()` to just implicitly transform the `options["west_cmd"]` into a list or there is any case that such a option will have a string with a space in it?

Reply:
it's because west could be in this format: `python3.8 -m west`